lang/python/pytz/Makefile

```diff
@@ -8,14 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pytz
-PKG_VERSION:=2018.6
+PKG_VERSION:=2018.7
 PKG_RELEASE:=1
 PKG_LICENSE:=MIT
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pytz
-PKG_HASH:=642253af8eae734d1509fc6ac9c1aee5e5b69d76392660889979b9870610a46b
+PKG_HASH:=31cb35c89bd7d333cd32c5f278fca91b523b0834369e757f4c5641ea252236ca
 
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
would `HOST_PYTHON_PACKAGE_BUILD_DEPENDS:="pytz==$(PKG_VERSION)"` also work ?
that uses the host pip installation mechanism ;
you only need to add this line before `include ../python-package.mk` and it should work

typically, using this variable/mechanism is useful when you want to add a `python3-pytz` variant or something like that; because you also have `HOST_PYTHON3_PACKAGE_BUILD_DEPENDS` and that simplifies the `Host/Compile` part quite a bit when building/installing for both Python & Python3

but, if there are no plans for Python3, this is also fine (i.e. the `Host/Compile`)

Reply:
Ok